site2/docs/io-elasticsearch-sink.md

```diff
@@ -16,11 +48,45 @@ The configuration of the ElasticSearch sink connector has the following properti
 |------|----------|----------|---------|-------------|
 | `elasticSearchUrl` | String| true |" " (empty string)| The URL of elastic search cluster to which the connector connects. |
 | `indexName` | String| true |" " (empty string)| The index name to which the connector writes messages. |
+| `schemaEnable` | Boolean | false | false | Turn on the Schema Aware mode. |
+| `createIndexIfNeeded` | Boolean | false | false | Manage index if missing. |
+| `maxRetries` | Integer | false | 1 | The maximum number of retries for elasticsearch requests. Use -1 to disable it.  |
+| `retryBackoffInMs` | Integer | false | 100 | The base time in milliseconds to wait when retrying an elasticsearch request. |
```

Comment:
```suggestion
| `retryBackoffInMs` | Integer | false | 100 | The base time to wait when retrying an ElasticSearch request (in milliseconds). |
```
Better to include units at the end w/ parenthese, which makes the info more clear. If agree, please update all.